lms/lmsweb/views.py

```diff
@@ -501,9 +507,13 @@ def _common_comments(exercise_id=None, user_id=None):
     Most common comments throughout all exercises.
     Filter by exercise id when specified.
     """
-    query = CommentText.filter(**{
-        CommentText.flake8_key.name: None,
-    }).select(CommentText.id, CommentText.text).join(Comment)
+    query = (
+        CommentText.select(CommentText.id, CommentText.text).join(Comment)
+        .join(User).join(Role).where(
+            CommentText.flake8_key.is_null(True),
+            Comment.commenter.role > Role.get_student_role().id,
```

Comment:
`role` doesn't have total ordering, and we don't wanna take the comments of banned users.

Check if role == administrator or role == staff